### ISSUE TITLE
Will it be finally open sourced?

### ISSUE THREAD

**Issue by iahung2** [cxasm/mywrite#2]
This is my question.

**Comment by cxasm** [comment]
Firstly, I am in China and due to some reasons, accessing GitHub is very slow. I submit a code or reply to a message, sometimes requiring more than 10 page refreshes. I can't even open it at all. China is in an isolated island in the global Internet. Additionally, using VPN casually is also illegal. So I can hardly do open source projects on GitHub.


Secondly, doing open source in China cannot support oneself and will face economic difficulties.


And once it becomes open source, everyone will plagiarize, making it even worse for developers, which is a unique feature of China.


I originally thought that if NDD were to be launched on Mac OS or become popular abroad, and could earn me living expenses through legitimate sales or sponsorship, then I would fully open up NDD. But currently, this goal is far from being achieved. Because Chinese developers face difficulties in legally receiving payments and promoting software overseas.


Finally, if NDD occupies a portion of the foreign market and I can earn legitimate business income through NDD, then I will fully open up NDD. I firmly believe that NDD will have a significant impact on similar expensive paid software, such as Subtext and UE.


In summary, currently only the basic version of Ndd can be open source.
